lib/datasource/common.ts

```diff
@@ -26,6 +26,8 @@ export interface Release {
 export interface ReleaseResult {
   sourceDirectory?: string;
   latestVersion?: string;
+  license?: string;
```

Comment:
Is this intended to be a standardized field, e.g. SPDX identifier? Or it could vary per-datasource?

Reply:
This is SPDX